bin/local-env/docker-compose.yml

```
@@ -37,6 +37,8 @@ services:
   mysql:
     image: mysql:5.7
     restart: always
+    ports:
+      - 3306:3306
```

Per my comment on the IB
```suggestion
      - 9306:3306
```